### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by DanielePalombo**
These changes are required to let to the bolt team to onboard their first customer.

### CODE REVIEW

**Review by MassimilianoLattanzio — APPROVED** [review]
LGTM

**Review by piyushswain — DISMISSED** [review]
I think we should remove the `bearer_token` field from the `solidus_bolt_bolt_configurations` table as well.

**Review by MassimilianoLattanzio — APPROVED** [review]

File: solidus_bolt.gemspec [hunk 1]
```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'coffee-rails'
   spec.add_dependency 'deface'
   spec.add_dependency 'httparty'
+  spec.add_dependency 'multi_json'
```
Comment (Naokimi) [inline solidus_bolt.gemspec:34]: not clear why we need the dependency

**Review by Naokimi — APPROVED** [review]

**Review by piyushswain — APPROVED** [review]
Looks good now

**Pull request merged**